tests/components/DrugOrderEntry/addForm/AddForm.test.jsx

```diff
@@ -232,5 +240,35 @@ describe('handleSubmitDrugForm() method', () => {
     expect(hhhSpy).toHaveBeenCalledTimes(1);
     expect(component.state.draftOrder).toEqual({ ...component.state.draftOrder, action: 'NOT_NEW', });
   });
+  it('displays a toast with message if a drug has been added successfully', () => {
+    const wrapper = getComponent();
+    wrapper.setProps({
+      ...wrapper.props(),
+      addDrugOrderReducer: {
+        errorMessage: '',
+        addedOrder: { id: 1, type: 'testorder' },
+        status: {
+          added: true,
+          error: false,
+        }
+      }
+    })
+    expect(global.toastrMessage).toEqual('Order Successfully Created')
```

Comment:
Please assert that `global.toastrMessage` is not equal to `Order Successfully Created` before setting the props.

Reply:
It doesn't make sense that way as there is no reason to assert when no action has been made, assertions are made to ensure that a mocked action would always return the expected result, just asserting at the beginning of a test-case without a preceding action happening gives no context on what the assertion is for.

Reply:
From the test description: `...displays a toast with a message if a drug has been added successfully...`, I believe this is what we want to test:
It is only when the drug is added(which you simulated with `wrapper.setPros({...})`) that the toast should appear with the message.

However, from the test implementation above, there is nothing that guarantees that `global.toastrMessage` does not alreay equals `Order Successfully Created` before you set the props. Adding an assertion to confirm that the `global.toastrMessage` does not contain `Order Successfully Created` before you set the props would give me enough confidence to believe that it is only after you set the props that the message appears.

Reply:
addressed